### PR TITLE
:memo: Condition and Family Module Drafts

### PR DESCRIPTION
**Pull request by aw3334**
# Motivation
We need module documentation for family and condition

# Approach
Starts a draft for each of these pages

### CODE REVIEW

File: input/pagecontent/module_family.md [hunk 1]
```diff
@@ -1 +1,21 @@
-### Family
+# Family
+
+## Overview
+This module describes family groups and family relationships, often used in genomic analyses to aid in identification of causal variants. We include several articulations of these: Family Study is an entity that describes a collection of participants in the context of a research study. Family Relationships describe the relationship of two Participants, and should be sufficient for generating a pedigree if the source information is present.
+
+This does not include specific methods for managing reports of “family history”. Those elements can either be encoded as specific “Participants” for which data are recorded as assertions and their family relationship to the Participant are recorded OR encoded as assertions from the Participant (eg, “family history of hypertension” or “Maternal history of hypertension”).
+
```
Comment (allisonheath) [inline input/pagecontent/module_family.md:7]: I think we should probably note awareness of the GA4GH efforts on family health history and KIN ontology: https://github.com/ga4gh/pedigree-fhir-ig https://github.com/ga4gh/pedigree_family_history_terminology and that part of the work to do is alignment between these efforts which are also anchored on being useful in genetic analysis.

File: input/pagecontent/module_phenotype_condition.md [hunk 2]
```diff
@@ -1 +1,28 @@
...
+
+## Conventions
+These entities describe phenotypes of Participants. The distinction is one of “record of asserted status” and “Details of positively asserted status”. We make these distinctions to support general use cases of “longitudinal, catch all records” and “curated summaries of features”. This distinction is focused on the interoperability utility over the “direct representation of reality”. There is overlap in these data; many assertions about status and onset might be coalesced into a single description of an affected status. In other studies, the “summary” may be derived from a single “assertion” and appear mostly duplicative.
+
+
+
+This distinction is similar to those in OMOP and FHIR, though it does not map precisely. Critically in OMOP and FHIR, Condition_occurrence and Condition are ALWAYS a positive assertion (though they may be wrong). “Assertions of history of disease” are observations in OMOP, eg, ICD9CM V-codes like “Personal history of malignant neoplasm of breast” go in Observation.
```
Comment (allisonheath) [inline input/pagecontent/module_phenotype_condition.md:19]: I'm wondering if calling it "Condition Module" is going to make people assume we're using the FHIR Condition resource under the hood? And that this piece is too buried in the versus being a bit up front that it's actually an Observation and/or that we're talking more generically about it versus the specific Condition resource?

**Review by allisonheath — CHANGES_REQUESTED** [review]
Some small comments

File: input/pagecontent/module_family.md (hunk 1 above)
Comment (calkinsh) [reply input/pagecontent/module_family.md:7]: @allisonheath I added some language here on line 8

File: input/pagecontent/module_phenotype_condition.md (hunk 2 above)
Comment (RobertJCarroll) [reply input/pagecontent/module_phenotype_condition.md]: One other thought on this: The "Condition Module" should ALSO have a condition profile IMO. The "Condition Summary" resource would help round this out. I think we want people looking for "Conditions" or looking for "Condition Resources" to come here to understand what we are doing.

File: input/pagecontent/module_family.md (hunk 1 above)
Comment (calkinsh) [reply input/pagecontent/module_family.md:7]: just documenting that i updated this language based on Allison's feedback

File: input/pagecontent/module_phenotype_condition.md (hunk 2 above)
Comment (calkinsh) [reply input/pagecontent/module_phenotype_condition.md]: I am going to merge the current state of these changes but open a ticket to add updates @RobertJCarroll is describing here.

**Review by allisonheath — APPROVED** [review]

**Pull request merged**